assets/templates/homepage/in-focus.tmpl

```diff
@@ -1,20 +1,20 @@
 <section>
-    <div class="col">
-        <h1 class="font-size--h2 font-size--30 margin-top--1">{{ localise "InFocus" .Language 1 }}</h2>
-    </div>
-    {{ range .Data.Featured }}
-        <section class="col col--md-23 col--lg-14 background--white margin-left-md--1 margin-bottom--3">
-            <article tabindex="0" class="tile tile__highlighted-content">
+    <h1 class="font-size--h2 font-size--30 margin-top--1">{{ localise "InFocus" .Language 1 }}</h2>
+    <div class="margin--0 flex stretch flex-wrap-wrap margin-bottom--3 content-space-between content-lg--flex-start">
+    {{ range $i, $v := .Data.Featured }}
+        <section class="tile col--md-23 col--lg-14 flex-basis-sm--full {{if ne $i 3}}margin-right-lg--1{{end}}">
+            <article tabindex="0" class="tile__highlighted-content">
```

Comment:
Noticed that the focus on the In Focus tiles is a bit different now:

**Before**:
![image](https://user-images.githubusercontent.com/23668262/93753593-4b2e2d00-fbf8-11ea-8e67-cb3eb139ade0.png)

**After**:
![image](https://user-images.githubusercontent.com/23668262/93753552-3d78a780-fbf8-11ea-8cfb-0f18ed921ed5.png)

The focus on Around The Ons tiles is the same, though; may be worth looking at if there's a class etc. that we've removed that was applying that focus style?